app/backend/create-server.ts

```diff
@@ -1,15 +1,17 @@
-const debug = require('debug')('seq:server');
-const http = require('http');
+import Debug from "debug";
+import { createServer as _createServer } from 'http';
+import dotenv from "dotenv";
 
-require('dotenv').config({ path: __dirname + '/../.env' });
+dotenv.config({ path: __dirname + '/../.env' });
 
+const debug = Debug("seq:server");
```

Comment:
Were there debug calls in this file as you were developing? Unlike console logs, properly configured debug statements do not need to be removed before committing.

Reply:
It was from the original server from where I [copied](https://github.com/button-inc/online-shop-form/blob/c080ce812973be1b64d81fb7d8ddb40919e8bff2/app/backend/create-server.js#L1) the code from. Would like to hear more about better debugs.